test/Tester/CancellationTests/CancellationTokenPassingTests.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.TestingHost;
+using UnitTests.GrainInterfaces;
+using Xunit;
+using UnitTests.Tester;
+
+namespace UnitTests.MembershipTests
+{
+    public class CancellationTokenPassingTests : HostedTestClusterPerTest
```

Comment:
For another non-native speaker this suggests the `CancellationToken` tests here should pass. Just a note. :)